modules/testsuite/shared-tests/src/test/java/org/jboss/test/ws/jaxws/jbws2150/JBWS2150TestCase.java

```diff
@@ -75,7 +76,8 @@
  * @author richard.opalka@jboss.com
  * @author alessio.soldano@jboss.com
  */
-@RunWith(Arquillian.class)
+@ExtendWith(ArquillianExtension.class)
+@Disabled("temporary disabled because of deployment is not injected in BeforeEach method call in junit5")
```

Comment:
Add TODO comment and later we can look at enable this test after the ARQ issue is fixed ?